### ISSUE TITLE
Getting Started with Binance

### ISSUE THREAD

**Issue by gabrieltemtsen** [stackup-dev/open-source-campaign#55]
# Name of Proposer
*Gabrieltemtsen*

# Campaign Name
*Entering the world of Binance.*

## Campaign Description
- This campaign seeks to instruct users on how to create Dapps and smart contracts on the bnb network as well as the infrastructure of the Binance Smart Chain.
- The campaign is appropriate for both new and experienced users.
- Since they have already learned about other mechanisms from previous campaigns (Algorand and Polkadot), Stackies should learn about this topic to get familiar with the proof of staked authority (PoSA) on the Binance smart chain.

## Number of Quests
2

## Quest 1 Summary
- Understanding blockchain and smart contract concepts 
- Hop on to Binance Academy
- intends to gain knowledge of blockchain technology and fundamental crypto concepts.*

## Quest 1 Learning Objectives
*Blockchain Terminologies.*
*Understanding many options that blockchain technology offers for developers.*

## Quest 1 Deliverables
*a quiz submission*

## Quest 2 Summary
- Building a smart contract on the Binance smart chain. 
- remix IDE, learn ETH, solidity
- Become knowledgeable about creating and deploying smart contracts.*

## Quest 2 Learning Objectives
*Built a smart contract.*
*Deployed a smart contract on the BNB network*

## Quest 2 Deliverables
*a screenshot of deployed smart contract*

**IMPORTANT:** You should have **Summary**, **Learning Objectives** and **Deliverables** sections for each quest you are suggesting. For example, if you have 3 quests, you should fill in this information for Quest 1 in the template above, then copy and paste the headings and fill up the information for Quest 2 and Quest 3 also.

**Comment by Marco-andana** [comment]
I agree with him, and I have suggestions for improvement to this request. More quiz quest about Binance. I suggest for a quest to make a Binance account too. The rest of them are good and just like what i want to say before. Please add this Stackup Teams :D

**Comment by omiklakhani** [comment]
omik

I agree with him, and I have suggestions for improvement to this request. More quiz quest about Binance. I suggest for a quest to make a Binance account too. The rest of them are good and just like what i want to say before. Please add this Stackup Teams :D

**Comment by onlybaba1234** [comment]
hansa


IMPORTANT: You should have Summary, Learning Objectives and Deliverables sections for each quest you are suggesting. For example, if you have 3 quests, you should fill in this information for Quest 1 in the template above, then copy and paste the headings and fill up the information for Quest 2 and Quest 3 also.
